### DISCUSSION TITLE
Set permissions when creating a new collection

### DISCUSSION THREAD

**Discussion by Loosie94** [directus/directus#20957] [Feature Requests]
### Summary

When creating a new collection, multiple default settings can already be set. 
The thing that cannot be set here are permissions. This is a thing that has to be set after creating of the collection, and can be forgotten. I think this is a missed opportunity and can solve frustration when trying to get the data of a collection from a web-application. 

### Basic Example

I would like to see the new tab on the create collection interface.
I have created an example image, using the same UI as the settings page for permissions:

![permissions-settings-collection-create](https://github.com/directus/directus/assets/13577640/49c4094e-c23b-43a7-bcac-44911f994e0c)

### Motivation

I think this could solve a lot of frustration of developers, and is a quicker way to create a new collection as you do not have to do this later on. 

### Detailed Design

See example at "Summery" and "Basic Example".

### Must Have:
- A Permissions tab
- B Settings per user role

### Drawbacks

I have no idea about the costs. A drawback can be that it has to work when the collection does not yet exist.

### Alternatives

A notice for giving the correct permissions after creating with a shortcut?

### Adoption Strategy

Maybe a plugin is another good way for this option. I am not sure how to adopt it.

### Unresolved Questions

I hope this is possible because it has to work when the collection does not yet exist.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
